conf/logstash/01-input.conf

```diff
@@ -1,5 +1,10 @@
 input {
-  beats {
-    port => 5044
-  }
-}
\ No newline at end of file
+    udp {
+        port => 10514
+        host => "::"
+        id => "udp_in"
+        receive_buffer_bytes => 1073741823
```

Comment:
Just found out that this could prevent logstash udp_in from working correctly, so I suggest to comment it out, cf. https://www.elastic.co/guide/en/logstash/current/plugins-inputs-udp.html#plugins-inputs-udp-receive_buffer_bytes